homeassistant/components/apple_tv/__init__.py

```diff
@@ -70,205 +70,257 @@ def ensure_list(value: Union[T, Sequence[T]]) -> Sequence[T]:
     extra=vol.ALLOW_EXTRA,
 )
 
-# Currently no attributes but it might change later
-APPLE_TV_SCAN_SCHEMA = vol.Schema({})
 
-APPLE_TV_AUTHENTICATE_SCHEMA = vol.Schema({ATTR_ENTITY_ID: cv.entity_ids})
+async def async_setup(hass, config):
+    """Set up the Apple TV integration."""
+    if DOMAIN not in config:
+        return True
+
+    for conf in config[DOMAIN]:
+        hass.async_create_task(
+            hass.config_entries.flow.async_init(
+                DOMAIN, context={"source": config_entries.SOURCE_IMPORT}, data=conf,
+            )
+        )
 
+    return True
 
-def request_configuration(hass, config, atv, credentials):
-    """Request configuration steps from the user."""
-    configurator = hass.components.configurator
 
-    async def configuration_callback(callback_data):
-        """Handle the submitted configuration."""
+async def async_setup_entry(hass, entry):
+    """Set up a config entry for Apple TV."""
+    identifier = entry.data[CONF_IDENTIFIER]
```

Comment:
config entries have a unique_id, you can use that instead, see hue or deconz

Reply:
I will change to that!